extensions/wikia/DesignSystem/services/DesignSystemGlobalFooterService.class.php

```diff
@@ -2,7 +2,14 @@
 
 class DesignSystemGlobalFooterService extends WikiaService {
 	public function index() {
-		$this->setVal( 'model', $this->getData() );
+		$model = $this->getData();
+
+		$this->setVal( 'model', $model );
+
+		//if ($model['is-wikia-org']) {
+		if (true) {
```

Comment:
Should probably swap this out for the commented out code above :)

Reply:
Also let's verify why it doesn't work on `mediawiki119`

Reply:
Yup, I'll do that once DR is complete :)